sdk/features/arm-features/README.md

```diff
@@ -1,11 +1,11 @@
 ## Azure FeatureClient SDK for JavaScript
 
-This package contains an isomorphic SDK (runs both in Node.js and in browsers) for FeatureClient.
+This package contains an isomorphic SDK (runs both in node.js and in browsers) for FeatureClient.
```

Comment:
This should be capitalized, i.e. `Node.js`. It has been fixed in code gen but I am waiting on @joheredi to come back from vacation and make a release, for now please update it manually:
```suggestion
This package contains an isomorphic SDK (runs both in Node.js and in browsers) for FeatureClient.
```